client/alloc_endpoint.go

```diff
@@ -62,8 +62,7 @@ func (a *Allocations) GarbageCollect(args *nstructs.AllocSpecificRequest, reply
 	}
 
 	if !a.c.CollectAllocation(args.AllocID) {
-		// Could not find alloc
-		return nstructs.NewErrUnknownAllocation(args.AllocID)
+		return fmt.Errorf("No such allocation on client or allocation not eligible for GC")
```

Comment:
The negations confused me a bit - having the comma improves readability a tiny bit:
```suggestion
		return fmt.Errorf("No such allocation on client, or allocation not eligible for GC")
```

Reply:
Good call. Much better.